src/composer-normalize.php

```diff
@@ -176,20 +184,47 @@ public function __construct(string $composerFile, TaskReportInterface $report, a
 
                     public function write(string $data, int $channel): void
                     {
+                        // strip ansi codes.
```

Comment:
I have no idea since when `--no-ansi` is available as previous versions did not output ANSI at all.

Therefore I choose this approach. If you know that it also works on legacy versions, we can skip this here and adjust the invocation instead.